localisation/Idea_Variation_ideas_l_english.yml

```diff
@@ -145,22 +145,24 @@ l_english:
  monarchie7_desc: "Our nobility will muster men to form militias when it comes to defending the homeland. \nEnable §GConscript Local Militia§W Estate Interaction"
  monarchie5: "Increase of Crown Authority"
  monarchie5_desc: "Increasing the crown authority will make it easier for us to govern the realm."
+
  republik0: "Republican"
  republik0_desc: "The Republican form of government rules in our country."
- republik7: "Roman History"
- republik7_desc: "By studying Roman history, we can learn from their state maintenance and their ability to quickly conquer and maintain new lands."
+ republik1: "Diplomatic Mission"
+ republik1_desc: "We need to play the diplomatic game to ensure our freedom. Because our way of life is both a chance and a threat to our neighbors."
  republik2: "Abolished Serfdom"
  republik2_desc: "We know from reading the accounts of Roman times that a citizen army of free men is far more willing to fight than those who are forced. We must make our peasantry fairly free and they will fight to defend their freedom."
  republik3: "Bill of Rights"
  republik3_desc: "By granting the citizens certain fundamental rights, we reduce the likelihood of a peasant uprising."
- republik5: "Republican Tradition"
- republik5_desc: "As a long standing republic, our government will more easily face and endure challenges. \nEnable §GPaying court to the Burghers§W Estate Interaction"
  republik4: "Free Subjects"
  republik4_desc: "Although giving a servant a damn good thrashing might be satisfying, it is not the best way to actually get them to work harder. Some natural philosopher has hit on the idea of making profit the prime motivation for people. If we replace monetary rents with payment in kind, our peasants will be incentivized to produce more."
- republik1: "Diplomatic Mission"
- republik1_desc: "We need to play the diplomatic game to ensure our freedom. Because our way of life is both a chance and a threat to our neighbors."
+ republik5: "The Power of a Diplomat"
```

Comment:
(Suggestion) As a running gag, whenever a function of an idea is changed, I've been using [alliteration](https://en.wikipedia.org/wiki/Alliteration) in individual Idea names when possible. In that vein, here's a list of suggested idea names for this:

- Dynamic Diplomat
- Effective Envoy
- Powerful Plenipotentiary (my current favorite)
- Convincing Conciliator
- Persuasive Peacemaker

... and so on.

Reply:
As long as it's written in English I'm happy.

Reply:
Is plenipotentiary even English at this point ? Although I like it for jokes

Reply:
Ha, it works well here doesn't it :)